Makefile

```diff
@@ -100,13 +100,6 @@ ifeq ($(GOOS),windows)
 	IS_EXE = ".exe"
 endif
 
-ifeq ($(GOARCH),amd64)
-	ARCHTAG ?= -amd64
-	ARCHTAG_NONE ?=
-else
-	ARCHTAG ?= -$(GOARCH)
-	ARCHTAG_NONE ?= -$(GOARCH)
-endif
 
 DEPLOYS=\
```

Comment:
I believe DEPLOYS should be removable with this change as well. You'll need to rename the .template files back into .yaml though.